app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -126,15 +132,14 @@ class WebViewFragment : Fragment(), NativePlayerHost {
     }
 
     private fun WebView.initialize() {
-        if (isOutdated()) { // Check WebView version
+        if (!appPreferences.ignoreWebViewChecks && isOutdated()) { // Check WebView version
             AlertDialog.Builder(requireContext()).apply {
                 setTitle(R.string.dialog_web_view_outdated)
                 setMessage(R.string.dialog_web_view_outdated_message)
-                setCancelable(false)
```

Comment:
I'm not sure it should be that easy. People will not read and/or accidentally dismiss the prompt..

Reply:
Adding 4 buttons requires a custom layout tho

Reply:
Maybe replace "close app" with "ignore"?

Reply:
Yes, good idea.